earthpy/plot.py

```diff
@@ -401,7 +401,15 @@ def plot_rgb(
 
 
 def hist(
-    arr, colors=["purple"], figsize=(12, 12), cols=2, bins=20, title=None
+    arr,
+    colors=["purple"],
+    figsize=(12, 12),
+    cols=2,
+    bins=20,
+    title=None,
+    xlabel="",
+    ylabel="",
+    **kwargs,
```

Comment:
i've added this but it's untested. we should test it using *range* if it works, then we can actually remove bins as an explicit parameter and just include it via kwargs